include/vcpkg/paragraphparser.h

```diff
@@ -30,8 +30,26 @@ namespace vcpkg
             return !missing_fields.empty() || !extra_fields.empty() || !expected_types.empty() ||
                    !other_errors.empty() || !error.empty();
         }
+
+        void format_to(LocalizedString& target) const;
+        LocalizedString format() const;
```

Comment:
This should learn to `to_string()` and reuse the common implementation macro for `fmt::formatter`.

Reply:
`to_string()` looses the information that is localized though.

Reply:
(And the _to version is here to make constructing the big blob of all of them more efficient, although it's probably not an inefficiency worth worrying about :) )